react-validated-input-component/my-app/src/ValidatedInput.css

```diff
@@ -0,0 +1,23 @@
+div {
+  display: flex;
+  flex-direction: column;
+  text-align: left;
+  width: 250px;
+  height: auto;
+}
+
+input {
+  max-width: 200px;
+}
+
+p {
+  line-height: 0;
+  color: #ff2424;
+}
+
+span {
+  width: 20px;
+  position: relative;
```

Comment:
Since the parent container is a flex div, you don't need to position this. We can take advantage of flexbox.

Reply:
Check out this handy guide! https://css-tricks.com/snippets/css/a-guide-to-flexbox/ I would bookmark it for future use!